modules/installation-osp-kuryr-known-limitations.adoc

```diff
@@ -11,11 +11,11 @@ Using {product-title} with Kuryr SDN has several known limitations.
 [id="openstack-general-limitations_{context}"]
 == {rh-openstack} general limitations
 
-{product-title} with Kuryr SDN does not support NodePort services. 
+{product-title} with Kuryr SDN does not support node port services. 
```

Comment:
I'm not involved with Kuryr, but I assume this means `Service` objects with `type=NodePort`; I'm not sure if a casual reader will understand that's what a _node port service_ is. I wouldn't if I didn't document SDN.